src/api/app/models/relationship.rb

```diff
@@ -50,10 +50,19 @@ class Relationship < ApplicationRecord
     joins(:role, :group).order('roles.title, groups.title')
   }
   scope :maintainers, lambda {
-    where(role_id: Role.hashed['maintainer'])
+    where(role: Role.hashed['maintainer'])
   }
+  # FIXME: This probably can be refactored to avoid instantiation inside the `map`
+  scope :for_maintainer_and_reviewer_roles, lambda {
+    where(role: Role.hashed['maintainer'])
```

Comment:
@danidoni this doesn't consider the `reviewer` role anymore

Reply:
But you for sure should change the name...

Reply:
But only for the reviewers of the target package of the request right? So if I add a review `by_package` for a package that is not the target of the request, the reviewer won't appear. I don't know if that is a use case

Reply:
Not sure, but but lets say the request targets package "apache2" and I add a review `by_package` for package `apache-rex` it will only show the reviewers for package `apache2`. If thats a real life scenario I would say we need to include the `reviewer` role in the scope.

Reply:
This is how it works now:
- It shows **only** reviewers from request target's package + project not maintainers
- It shows **only maintainers** from explicit reviews (by package or by project) not reviewers

Is this how it should work?

Reply:
This is how it works now:
- It shows **only** reviewers from request target's package + project not maintainers
- It shows **only maintainers** from explicit reviews (by package or by project) not reviewers

Is this how it should work?

Reply:
As discussed offline, for now we just consider the maintainer role for the review `by_package`